MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -268,3 +276,20 @@ extension PassiveLocationDataSource {
         public static let roadNameKey: NotificationUserInfoKey = .init(rawValue: "roadName")
     }
 }
+
+extension MapboxNavigationService {
+    /**
+     Keys in the user info dictionaries of various notifications posted by instances of `NavigationService`.
+     */
+    public struct NotificationUserInfoKey: Hashable, Equatable, RawRepresentable {
+        public typealias RawValue = String
+        public var rawValue: String
+        public init(rawValue: String) {
+            self.rawValue = rawValue
+        }
+    }
+
+    /**
+     A key in the user info dictionary of a `Notification.Name.locationAuthorizationDidChange` notification. The corresponding value is a `Int` object the current location authorization setting. */
```

Comment:
It’s technically an `Int` when we put it in the dictionary, due to our internal workaround, but the developer would be expected to cast it to a `CLAccuracyAuthorization`.

If the developer needs to similarly work around the lack of `CLAccuracyAuthorization` on iOS 13, they’d conditionally implement an enumeration on iOS 13 that would match a typedef on iOS 14. But documenting that workaround is probably outside the scope of the documentation here, since the workaround requires a modicum of Objective-C code that might not be appropriate for every application.

```suggestion
     A key in the user info dictionary of a `Notification.Name.locationAuthorizationDidChange` notification. The corresponding value is a `CLAccuracyAuthorization` indicating the current location authorization setting. */
```